packages/cli/lib/lib/entry.js

```diff
@@ -3,6 +3,8 @@ import { h, render } from 'preact';
 if (process.env.NODE_ENV==='development') {
 	// enable preact devtools
 	require('preact/debug');
+	// eslint-disable-next-line no-undef
+	navigator.serviceWorker.register(__webpack_public_path__ + 'sw_debug.js');
```

Comment:
should we unregister any existing SW in this case?

Reply:
makes sense... and just in case someone wants to add their own SW at dev time may be add a flag to disable this?

cc: @harshitkumar31